src/app/maps/m6/infobar/infobar.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Output } from '@angular/core';

```

Comment:
This component is not used anywhere

Reply:
we made a copy but we forgot to remove it. i remove it